### ISSUE TITLE
README error, should be: mkdir /exports/nfs-direct, not /exports/shared

### ISSUE THREAD

**Issue by djarotech** [kmodules/third-party-tools#6]
Readme in question: https://github.com/appscode/third-party-tools/blob/master/storage/nfs/README.md
```
At first, let's create nfs-direct folder inside /exports directory of NFS server.

$ kubectl exec -n storage nfs-server-f9c6cbc7f-n85lv mkdir /exports/shared
Verify that directory has been created successfully,

$ kubectl exec -n storage nfs-server-f9c6cbc7f-n85lv ls /exports
index.html
nfs-direct
```
should be changed to:
```
At first, let's create nfs-direct folder inside /exports directory of NFS server.

$ kubectl exec -n storage nfs-server-f9c6cbc7f-n85lv mkdir /exports/nfs-direct
Verify that directory has been created successfully,

$ kubectl exec -n storage nfs-server-f9c6cbc7f-n85lv ls /exports
index.html
nfs-direct
```

**Comment by hossainemruz** [comment]
@danielpygo Thank you for reporting it. We have fixed it in #8 .

**Comment by djarotech** [comment]
Sure thing, and oh nice !
